### DISCUSSION TITLE
IBAX AÇILIM FİYATI NE OLACAK SİZCE ?

### DISCUSSION THREAD

**Discussion by harunkursun23** [IBAX-io/go-ibax#1239] [Q&A]
IBAX AÇILIM FİYATI NE OLACAK SİZCE PUANA KARŞILIK NE KADAR VERİLECEK 1 IBXC 10 PUANMI YOKSA 100 PUANMI

**Comment by hizen1984** [comment]
please wait , :) big project

**Comment by erhanqwerty** [comment]
1ile 30 usc (amerikan centi) arası açılış olabilir

**Comment by sedat2855** [comment]
1 USDT

**Comment by hizen1984** [comment]
10 usdt , big proje
________________________________
Gönderen: sedat2855 ***@***.***>
Gönderildi: 28 Ocak 2022 Cuma 19:16
Kime: IBAX-io/go-ibax ***@***.***>
Bilgi: hizen1984 ***@***.***>; Comment ***@***.***>
Konu: Re: [IBAX-io/go-ibax] IBAX AÇILIM FİYATI NE OLACAK SİZCE ? (Discussion #1239)


1 USDT

—
Reply to this email directly, view it on GitHub<https://github.com/IBAX-io/go-ibax/discussions/1239#discussioncomment-2067644>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXOP657FR4O7PRT3WW33M3TUYK6MHANCNFSM5NA5IQVQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by Abdullahoda** [comment]
0.13 dolar açılış fiyatı olucak dediler  puan sıralamasına göre ilk 1000 kişiye 500 ibax geri kalanlar esıt sekılde bölüşecekler.
